### ISSUE TITLE
Internal error on specific file

### ISSUE THREAD

**Issue by Seluj78** [psf/black#1171]
**Describe the bug**
Internal error of black (produced bad code)

**To Reproduce** Steps to reproduce the behavior:

1. Take this code

```python
import rsa
import json
# import pytz
import requests
import base64

# from datetime import datetime

from flask import Blueprint, render_template, flash, url_for, redirect, session

from vbackoffice.utils.decorators import login_required

from vbackoffice.forms.authentication import (
    LoginForm,
    # ForgotPasswordForm,
    # NewPasswordForm,
)

from vbackoffice.config import API_URL, pubkey

# from werkzeug.security import generate_password_hash
```

2. Run _Black_ on it with these arguments

```bash
black FILE.py
```

3. See error

```bash
(venv) ➜  /tmp black test.py
error: cannot format test.py: INTERNAL ERROR: Black produced invalid code: invalid syntax (<unknown>, line 18). Please report a bug on https://github.com/ambv/black/issues.  This invalid output might be helpful: /var/folders/_3/c77wmv3919n_gs69s5f2wms40000gn/T/blk_tev7lxs9.log
All done! 💥 💔 💥
1 file failed to reformat.
```

- Version: `19.10b`
- OS and Python version: macOS 10.14.6 / Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 16:52:21) 

**Does this bug also happen on master?** 

Doesn't look like it

**Comment by JelleZijlstra** [comment]
I can't reproduce this on master or 19.10b0. I believe I fixed this bug after the release of 19.3b0.

**Comment by Seluj78** [comment]
Weird, alright thanks ! I'll let you know if it happens again !
